README.md

```diff
@@ -14,3 +14,6 @@ Features
 - Serverless design has low cost, low operational overhead, and scales effortlessly
 
 Ready? Check out the [docs!](https://twilio-labs.github.io/socless/)
+
+
+Join our community Slack workspace @ https://join.slack.com/t/socless/shared_invite/enQtODA3ODEzNzcwNDgxLTBiYjVjYjI4ODI4YTY5YzM4OWRlYjQ1Yzg4M2EzMGUzMGMyYThlN2U5NTI5OWIwZWE1ZTcwNjA2MjgyZDRmMjg
```

Comment:
Recommend making this a markdown inline like e.g [community Slack workspace](join.slack.com/....)

Reply:
done!